docs/breez-sdk/src/guide/issuing_tokens.md

```diff
@@ -14,10 +14,14 @@ The Breez SDK provides a specialized Token Issuer interface for managing custom
 
 Create a custom token with configurable parameters. Define the decimal precision, max supply and if the token can be frozen.
 
-**Note:** Token creation is limited to one token per issuer wallet
-
 {{#tabs issuing_tokens:create-token}}
 
+### Creating multiple tokens
+
+Token creation is limited to one token per issuer wallet. If you need to create and then manage more than one token using the same mnemonic, we recommend using different account numbers when initializing the SDK.
```

Comment:
Why do we need to document this? Isn't it the same as multiple sdk instances where each instance actually mapped to a different spark user?